### ISSUE TITLE
column drilldown with http request

### ISSUE THREAD

**Issue by Bclayson** [pablojim/highcharts-ng#341]
If I try to set the drilldown series (in the 'options' section) inside a restangular/http promise with a column chart the chart will not draw. If I click the category name the drilldown data will appear. If I click the 'back' button the chart WILL draw and then it works fine. 

This ONLY is an issue with column/bar charts, and it ONLY has this problem when the data is set from an AJAX call. If i change the chart to a pie, the problem goes away. If I add the drilldown data statically the problem goes away. I'm at my wits end.

**Comment by Bclayson** [comment]
More info. I started to create a Plunkr by forking another Plunkr for convenience, and the problem went away! On closer inspection it looks that like the plunkr I was using uses an older version of highcharts-ng.  

So I created this plunkr with both version added. http://plnkr.co/edit/ga8QhUUfeDfy6U33YFr4?p=preview

If you go to index.html and comment out highcharts-ng.js and uncomment highcharts-ng-0.0.8.js You'll get the error I mentioned. 

Hopefully this can get fixed and I can use the newest version without issues. Thanks!
